core/tests/webdriverio_utils/workflow.js

```diff
@@ -226,11 +226,11 @@ var _addExplorationRole = async function(roleName, username) {
 };
 
 var addExplorationManager = async function(username) {
-  await _addExplorationRole('Manager', username);
+  await _addExplorationRole('Manager (can edit permissions)', username);
```

Comment:
Why  (can edit permissions)? Ditto elsewhere?

Reply:
As mentioned previously `selectByVisibleText` doesn't work if we do not pass the exact text that is present inside the `<option>` tag, so I am passing the exact string that we actually have in frontend.